data/Sally.txt

```diff
@@ -0,0 +1,5 @@
+T | 1 | read book
```

Comment:
Perhaps you can consider adding data files to `.gitignore`. Otherwise users who clone your repo may end up with tasks loaded that they did not add.